quant.install

```diff
@@ -145,3 +221,10 @@ function quant_update_7002() {
   variable_set('quant_queue_hash_method', 'sha256');
   db_create_table('quant_queue', drupal_get_schema_unprocessed('quant', 'quant_queue'));
 }
+
+/**
+ * Add the `quant_log` table.
+ */
+function quant_update_7003() {
+  db_create_table('quant_log', drupal_get_schema_unprocessed('quant', 'quant_log'));
```

Comment:
I think we also need to enable views in this upgrade too.

Reply:
Applying the upgrade without adding views doesn't cause issues however means the log page isn't available.